bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/utils/StringUtils.java

```diff
@@ -40,10 +40,11 @@
      * Returns the human-readable string value of Ballerina values.
      * @param strand  the strand in which the value reside
      * @param value The value on which the function is invoked
-     * @param type The type of value on which the function is invoked
      * @return String value of the value
      */
-    public static String getStringValue(Strand strand, Object value, BType type) {
+    public static String getStringValue(Strand strand, Object value) {
+        BType type = TypeChecker.getType(value);
```

Comment:
lets first check if the value is null then return "", otherwise check the type

Reply:
@gimantha fixed.